.github/workflows/ci.yml

```diff
@@ -46,9 +47,82 @@ jobs:
           PG_OPERATOR_VERSION: 2.5.0
           PERCONA_VERSION_SERVICE_URL: https://check-dev.percona.com/versions/v1
 
+  integration-tests:
```

Comment:
I was expecting a step to run `kind create cluster ...`, but that's not there. When does the kind cluster get created? 🤔

Reply:
This is handled internally by kuttl if you set the `StartKIND` field.
https://github.com/percona/everest-operator/blob/765cb75a70909d21cf1c5175f2186abb86a55acb/tests/integration/kuttl-features.yaml#L5